pkg/scd/operational_intents_handler.go

```diff
@@ -463,36 +476,36 @@ func (a *Server) PutOperationalIntentReference(ctx context.Context, manager stri
 
 		var sub *scdmodels.Subscription
 		if subscriptionID.Empty() {
-			// Create implicit Subscription
-			if params.NewSubscription == nil || params.NewSubscription.UssBaseUrl == "" {
-				return stacktrace.NewErrorWithCode(dsserr.BadRequest, "Missing new_subscription or uss_base_url in new_subscription")
-			}
-			if !a.EnableHTTP {
-				err := scdmodels.ValidateUSSBaseURL(string(params.NewSubscription.UssBaseUrl))
-				if err != nil {
-					return stacktrace.PropagateWithCode(err, dsserr.BadRequest, "Failed to validate USS base URL")
+			// Create implicit Subscription if the implicit subscription params are set:
+			// (for situations where these params are required, an error will have been returned earlier)
```

Comment:
I'm not clear on what this comment means -- is it supposed to be "for situations where these params are required, an error will have been returned earlier *if the params were not provided*"?

Reply:
Rephrased and expanded the comment.